### PR TITLE
APP-4343 Added keyboard navigation on close icon dropdown

### PR DESCRIPTION
**Pull request by symphony-annaLlorens**
## Description
- Added keyboard navigation on close icon dropdown
- Added aria label 
Demo:

https://user-images.githubusercontent.com/68587873/129881619-5a71e82c-2d68-44d0-b369-5e658d3c7686.mp4

### CODE REVIEW

File: src/components/dropdown/CustomRender.tsx [hunk 1]
```diff
@@ -125,9 +125,11 @@ export const DropdownIndicator = (props: any) => {
     </components.DropdownIndicator>;
 };
 
-export const ClearIndicator = (props: any) =>
+export const ClearIndicator = (props: any) => 
   <components.ClearIndicator {...props}>
-    <Icon iconName="cross-round"/>
+    <div className="tk-select__close-icon" onKeyPress={props.clearValue} tabIndex={0}>
```
Comment (AliciaSymphony) [inline src/components/dropdown/CustomRender.tsx:130]: What is the use of "tk-select__close-icon" ?

Comment (symphony-annaLlorens) [reply src/components/dropdown/CustomRender.tsx:130]: These styles come from UItoolkit-styles.

Comment (AliciaSymphony) [reply src/components/dropdown/CustomRender.tsx:130]: But I can't find it anywhere on UI Toolkit-styles
image
image

File: src/components/dropdown/CustomRender.tsx [hunk 2]
```diff
@@ -43,12 +43,12 @@ export const DefaultOptionRenderer = (props: any) => {
...
   ) : <>
     {OptionRenderer ? 
-      <div className="tk-option">
+      <div className="tk-option" role="option">
         <components.Option {...props}>
           <OptionRenderer {...rendererProps} />
         </components.Option>
       </div>
-      : <div className="tk-option">
+      : <div className="tk-option" role="option">
```
Comment (AliciaSymphony) [inline src/components/dropdown/CustomRender.tsx:51]: 👍

File: src/components/dropdown/CustomRender.tsx (hunk 1 above)
Comment (AliciaSymphony) [reply src/components/dropdown/CustomRender.tsx:130]: Now I see https://github.com/SymphonyPlatformSolutions/symphony-bdk-ui-toolkit-styles/pull/125 open so I understand :)

**Review by symphony-jean-michael — APPROVED** [review]
LGTM

**Review by AliciaSymphony — APPROVED** [review]

**Pull request merged**